integration-tests/aws/src/main/java/org/apache/camel/quarkus/component/aws/CamelRoute.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.quarkus.it.camel.aws;
+package org.apache.camel.quarkus.component.aws;
```

Comment:
I vote for keeping `it` somewhere in the integration test package names. `org.apache.camel.quarkus.component.aws.it` or `org.apache.camel.quarkus.it.component.aws` or some other variant. This is to avoid any potential naming clashes with the the platform code.

Reply:
The reason I do usually keep the same name is to be able to access package private methods/classes for testing purpose, I can change it but I propose to do it later

Reply:
> be able to access package private methods/classes for testing purpose

Yes, that's normal and OK for unit tests. OTOH integration tests should simulate how our end users use camel quarkus. We should not assume that they access non-public APIs.

Reply:
That's partially true as we do not have any unit test for the native image side of things